firmware-util.sh

```diff
@@ -37,31 +40,59 @@ else
 	export CURL="curl"
 fi
 
-#get support scripts
-echo -e "\nDownloading supporting files..."
-rm -rf firmware.sh >/dev/null 2>&1
-rm -rf functions.sh >/dev/null 2>&1
-rm -rf sources.sh >/dev/null 2>&1
-$CURL -sLO ${script_url}firmware.sh
-rc0=$?
-$CURL -sLO ${script_url}functions.sh
-rc1=$?
-$CURL -sLO ${script_url}sources.sh
-rc2=$?
-if [[ $rc0 -ne 0 || $rc1 -ne 0 || $rc2 -ne 0 ]]; then
-	echo -e "Error downloading one or more required files; cannot continue"
-	exit 1
+if [ ! -d "$script_dir/.git" ] || ! grep -qF 'mrchromebox-scripts.git' "$script_dir/.git/config"; then
```

Comment:
this seems specific to your naming of the forked repo, can we just drop the 2nd part? Why wouldn't the .git directory be a sufficient check?

Reply:
No problem, removed! I just thought slapping more "mrchromebox" everywhere so it would look more appealing to you :D